data/pools/transactionPool.go

```diff
@@ -738,15 +739,21 @@ func (pool *TransactionPool) recomputeBlockEvaluator(committedTxIds map[transact
 				pool.statusCache.put(tx, err.Error())
 			}
 
-			switch err.(type) {
+			switch err := err.(type) {
 			case *ledgercore.TransactionInLedgerError:
 				asmStats.CommittedCount++
 				stats.RemovedInvalidCount++
 			case transactions.TxnDeadError:
-				asmStats.InvalidCount++
+				if proto.MaxTxnLife == uint64(err.LastValid-err.FirstValid) {
+					asmStats.ExpiredMaxLifeCount++
```

Comment:
This can be misleading, since we don't know if `err.LastValid-err.FirstValid` is a small enough to make an impact, say less than 10, or if it is `proto.MaxTxnLife-1`.
I think it will be better to check if it is less than 50, than compare to `proto.MaxTxnLife`.

Reply:
I agree that this is odd, I just didn't want to introduce an arbitrary magic number since any number is really arbitrary. Perhaps more useful would be an average transaction lifespan? Could implement it as running average by keeping running sum of lifetimes of expired txns?

Reply:
I made the change to 20. We can see what stats this comes out to and further tune when removing old stats.